core/toolbox/collapsible_category.ts

```diff
@@ -200,6 +200,7 @@ export class CollapsibleToolboxCategory
       this.subcategoriesDiv_!.style.display = 'none';
       this.closeIcon_(this.iconDom_);
     }
+    this.parentToolbox_.getFlyout()?.setVisible(isExpanded);
```

Comment:
I believe we only want to do this when `isExpanded === false`! Otherwise when we expand a category, it opens the last open flyout, which may not even be contained within the collapsible category.